README.md

```diff
@@ -1,6 +1,32 @@
 # Perseus demo energy provider
 
-Emulates authentication and resource api endpoints for the Perseus demo. Authentication is built on top of [Ory Hydra](https://www.ory.sh).
+This repository contains two apps that demonstrate securing a data endpoint using the Fapi standard, in a way that is compliant with the [Perseus trust framework](https://ib1.org/perseus/)
```

Comment:
* I've always seen all-caps "FAPI"
* "mockup smart meter data endpoint" (so people get why it's called perseus-demo-energy)
* Perseus Scheme rather than TF